gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/spi/DefaultPubSubRpc.java

```diff
@@ -89,6 +93,56 @@ protected ExecutorFactory executorFactory() {
     }
   }
 
+  private static final class PullFutureImpl implements PullFuture {
```

Comment:
Better [SimpleForwardingListenableFuture](https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/util/concurrent/ForwardingListenableFuture.SimpleForwardingListenableFuture.html) so we can avoid casting `delegate()`. Fixed in 752fab0.